CMakeLists.txt

```diff
@@ -173,6 +173,12 @@ elseif(${S2_SOURCE} STREQUAL "SYSTEM")
   endif()
 endif()
 
+# this might be needed since s2geometry includes it in general but not for any
+# target explicilty?
```

Comment:
Agreed that comment was not very helpful (conda builds will be fine I think, the issue here was for Python wheels built and uploaded on PyPI via CI but we'll need to streamline the build process anyway).

> Just so that I don't remove that by accident again!

No worries!